masonry/src/widget/checkbox.rs

```diff
@@ -94,16 +92,15 @@ impl Widget for Checkbox {
                 accesskit::Action::Default => {
                     self.checked = !self.checked;
                     ctx.submit_action(Action::CheckboxChecked(self.checked));
-                    ctx.request_paint();
-                    ctx.request_accessibility_update();
+                    ctx.request_render();
                 }
                 _ => {}
             }
         }
     }
 
     fn on_status_change(&mut self, ctx: &mut LifeCycleCtx, _event: &StatusChange) {
-        ctx.request_paint();
+        ctx.request_render();
```

Comment:
Hmm, should this be the default implementation of `on_status_changed`?

Reply:
Maybe. I'd rather not have a default implementation at all for now.